docs/experimentConfigReadme.md

```diff
@@ -96,6 +96,7 @@ The following configuration only applies to player-bound parametric targets.
 
 * `speed` is a vector indictating the minimum ([0]) and maximum ([1]) speeds in angular velocity (in deg/s)
 * `distance` is the distance to this target (in meters)
+* `symmetricEcc` is a boolean flag indicating whether the target spawn eccentricities are to be considered in all 4 directions or not
```

Comment:
I suggest improving this description text, possibly including an example. Something along the lines of:

>When `True` eccentricities are assumed symmetric about 0° (the central view direction) and should always be >0. When `False` the eccentricities can be signed, with positive eccentricities indicate values to the right (azim) and up (elev) of the central view direction.